src/fidesops/models/privacy_request.py

```diff
@@ -158,6 +160,9 @@ class PrivacyRequest(Base):  # pylint: disable=R0904
         backref="privacy_requests",
     )
 
+    cancel_reason = Column(String)
```

Comment:
Seems like [the `sqlalchemy.types.Text` class](https://docs.sqlalchemy.org/en/14/core/type_basics.html#sqlalchemy.types.Text) might be more what we're looking for here? Otherwise we might want to also specify `length=...` so this will work fine if anyone recreates the migrations ([it's required for `CREATE TABLE ...` statements](https://docs.sqlalchemy.org/en/14/core/type_basics.html#sqlalchemy.types.String.params.length))

Reply:
OK I added a length to this particular field.